examples/Yolo-object-detection/README.rst

```diff
@@ -0,0 +1,58 @@
+FEDn Project: 'Yolo-object-detection'
```

Comment:
Can you expand in the readme with more information on what this example is about, maybe include some images etc to explain the ML task being solved, etc.